File: Em C/1009.c
#include <stdio.h>
 
int main() {
 
    char nome[20];
    double slfixo,tvendas, total;	
    
    scanf("%s",nome);
    scanf("%lf",&slfixo);
    scanf("%lf",&tvendas);
	tvendas=tvendas*0.15;
	total=slfixo+tvendas;
	printf("TOTAL = R$ %.2lf\n",total);
	
 
    return 0;
}

File: Em Ruby/1009.rb
nome = gets.chomp
slfixo = gets.chomp.to_f
tvendas = gets.chomp.to_f
	tvendas=tvendas*0.15;
	total=slfixo+tvendas;
	total=format("%.2f",total)
  puts "TOTAL = R$ #{total}"
